### DISCUSSION TITLE
Failed to load PostCSS config (breeze)

### DISCUSSION THREAD

**Discussion by khantseithu** [laravel/framework#47628] [General]
I'm following along laravel bootcamp in gitpod environment with laravel sail (docker option). 

In this command : php artisan breeze:install blade

Having this error:
![image](https://github.com/laravel/framework/assets/100980146/c75307e6-b072-4e4d-ae3a-aa4239ce2464)

Not loading styling properly on register and login pages.

Laravel Bootcamp: https://bootcamp.laravel.com/blade/installation

**Comment by datlechin** [comment]
Try to add `"type": "module"` in `package.json` file
